src/utils/ListOfWords.ts

```diff
@@ -1,220 +1,220 @@
   "absurd",
-  "abruptly",
-  "abyss",
-  "affix",
-  "askew",
-  "avenue",
-  "awkward",
-  "axiom",
-  "azure",
-  "bagpipes",
-  "bandwagon",
-  "banjo",
-  "bayou",
-  "beekeeper",
-  "bikini",
-  "blitz",
-  "blizzard",
-  "boggle",
-  "bookworm",
-  "boxcar",
-  "boxful",
-  "buckaroo",
-  "buffalo",
-  "buffoon",
-  "buxom",
-  "buzzard",
-  "buzzing",
-  "buzzwords",
-  "caliph",
-  "cobweb",
-  "cockiness",
-  "croquet",
-  "crypt",
-  "curacao",
-  "cycle",
-  "daiquiri",
-  "dirndl",
-  "disavow",
-  "dizzying",
-  "duplex",
-  "dwarves",
-  "embezzle",
-  "equip",
-  "espionage",
-  "euouae",
-  "exodus",
-  "faking",
-  "fishhook",
-  "fixable",
-  "fjord",
-  "flapjack",
-  "flopping",
-  "fluffiness",
-  "flyby",
-  "foxglove",
-  "frazzled",
-  "frizzled",
-  "fuchsia",
-  "funny",
-  "gabby",
-  "galaxy",
-  "galvanize",
-  "gazebo",
-  "giaour",
-  "gizmo",
-  "glowworm",
-  "glyph",
-  "gnarly",
-  "gnostic",
-  "gossip",
-  "grogginess",
-  "haiku",
-  "haphazard",
-  "hyphen",
-  "iatrogenic",
-  "icebox",
-  "injury",
-  "ivory",
-  "ivy",
-  "jackpot",
-  "jaundice",
-  "jawbreaker",
-  "jaywalk",
-  "jazziest",
-  "jazzy",
-  "jelly",
-  "jigsaw",
-  "jinx",
-  "jiujitsu",
-  "jockey",
-  "jogging",
-  "joking",
-  "jovial",
-  "joyful",
-  "juicy",
-  "jukebox",
-  "jumbo",
-  "kayak",
-  "kazoo",
-  "keyhole",
-  "khaki",
-  "kilobyte",
-  "kiosk",
-  "kitsch",
-  "kiwifruit",
-  "klutz",
-  "knapsack",
-  "larynx",
-  "lengths",
-  "lucky",
-  "luxury",
-  "lymph",
-  "marquis",
-  "matrix",
-  "megahertz",
-  "microwave",
-  "mnemonic",
-  "mystify",
-  "naphtha",
-  "nightclub",
-  "nowadays",
-  "numbskull",
-  "nymph",
-  "onyx",
-  "ovary",
-  "oxidize",
-  "oxygen",
-  "pajama",
-  "peekaboo",
-  "phlegm",
-  "pixel",
-  "pizazz",
-  "pneumonia",
-  "polka",
-  "pshaw",
-  "psyche",
-  "puppy",
-  "puzzling",
-  "quartz",
-  "queue",
-  "quips",
-  "quixotic",
-  "quiz",
-  "quizzes",
-  "quorum",
-  "razzmatazz",
-  "rhubarb",
-  "rhythm",
-  "rickshaw",
-  "schnapps",
-  "scratch",
-  "shiv",
-  "snazzy",
-  "sphinx",
-  "spritz",
-  "squawk",
-  "staff",
-  "strength",
-  "strengths",
-  "stretch",
-  "stronghold",
-  "stymied",
-  "subway",
-  "swivel",
-  "syndrome",
-  "thriftless",
-  "thumbscrew",
-  "topaz",
-  "transcript",
-  "transgress",
-  "transplant",
-  "triphthong",
-  "twelfth",
-  "twelfths",
-  "unknown",
-  "unworthy",
-  "unzip",
-  "uptown",
-  "vaporize",
-  "vixen",
-  "vodka",
-  "voodoo",
-  "vortex",
-  "voyeurism",
-  "walkway",
-  "waltz",
-  "wave",
-  "wavy",
-  "waxy",
-  "wellspring",
-  "wheezy",
-  "whiskey",
-  "whizzing",
-  "whomever",
-  "wimpy",
-  "witchcraft",
-  "wizard",
-  "woozy",
-  "wristwatch",
-  "wyvern",
-  "xylophone",
-  "yachtsman",
-  "yippee",
-  "yoked",
-  "youthful",
-  "yummy",
-  "zephyr",
-  "zenith",
-  "zigzag",
-  "zigzagging",
-  "zilch",
-  "zipper",
-  "zodiac",
-  "zombie",
-  "zillion",
-  "zaftig",
+  // "abruptly",
```

Comment:
Maybe use process.env.NODE_ENV === 'production' to change the word table during debugging only?